### PR TITLE
Update custom-style API doc

### PR DESCRIPTION
**Pull request by arthurevans**
Fixes https://github.com/Polymer/docs/issues/2266.

I didn't test this through the docs generation. Handle with care.

### CODE REVIEW

File: lib/elements/custom-style.html [hunk 1]
```diff
@@ -19,17 +19,43 @@
...
+   * advantage of [shady DOM](https://github.com/webcomponents/shadycss) shims
+   * for style encapsulation, custom properties, and custom mixins.
    *
    * - Document styles defined in a custom-style are shimmed to ensure they
    *   do not leak into local DOM when running on browsers without native
    *   Shadow DOM.
-   * - Custom properties used by Polymer's shim for cross-scope styling may
-   *   be defined in an custom-style. Use the :root selector to define custom
-   *   properties that apply to all custom elements.
+   * - Custom properties can be defined in an custom-style. Use the `html` selector
```
Comment (katejeffreys) [inline lib/elements/custom-style.html]: >Custom properties can be defined in an custom-style.

Should be 

Custom properties can be defined in a custom-style.

Also.. sometimes it is custom-style and other times `<custom-style>`. I am sure there is a reason, just wondering what it is :)

File: lib/elements/custom-style.html [hunk 2]
```diff
@@ -19,17 +19,43 @@
...
    * - Document styles defined in a custom-style are shimmed to ensure they
    *   do not leak into local DOM when running on browsers without native
    *   Shadow DOM.
-   * - Custom properties used by Polymer's shim for cross-scope styling may
-   *   be defined in an custom-style. Use the :root selector to define custom
-   *   properties that apply to all custom elements.
+   * - Custom properties can be defined in an custom-style. Use the `html` selector 
+   *   to define custom properties that apply to all custom elements.
+   * - Custom mixins can be defined in a custom-style, if you import the optional 
+   *   [apply shim](https://github.com/webcomponents/shadycss#about-applyshim)(`shadycss/apply-shim.html`).
```
Comment (katejeffreys) [inline lib/elements/custom-style.html]: >[apply shim](https://github.com/webcomponents/shadycss#about-applyshim)(`shadycss/apply-shim.html`).

Needs a space? i.e

[apply shim](https://github.com/webcomponents/shadycss#about-applyshim)
 (`shadycss/apply-shim.html`).

**Comment by katejeffreys** [comment]
Tested by following this process: https://www.polymer-project.org/2.0/docs/tools/documentation#create-page

Looks mostly great

**Comment by arthurevans** [comment]
Thanks for the fixes @katejeffreys—also for cluing me in that I could just review the changes using `polymer analyzer`/`polymer serve`! Fixed spacing and used `<custom-style>` consistently instead of switching back and forth.

**Pull request merged**